moviesTime/Network/DetailsMovie.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  DetailsMovie.swift
+//  moviesTime
+//
+//  Created by Jose Eduardo on 31/07/21.
+//
+
+/*
+    "adult":false,
```

Comment:
Don't leave comments in the code to show examples of the data, remove it before uploading any code to the repo